packages/design-system-react/jest.config.js

```diff
@@ -21,6 +21,7 @@ module.exports = merge(baseConfig, {
     'src/components/Icon/icons/', // Auto-generated icons
     'scripts/create-component/ComponentName/', // component template
     '\\.constants\\.(ts|js)$', // ignore all .constants.ts and .constants.js files
+    '\\.dev\\.(ts|js)$', // ignore all .constants.ts and .constants.js files
```

Comment:
Would it make sense to ignore these files from the `tsconfig.ts` so they aren't included in the build? I'm not sure if it matters if they're ignored from coverage or not they could